DOCUP-5978: Documented max_partitions_to_read (#19954)
ClickHouse/ClickHouse: docs/en/engines/table-engines/mergetree-family/mergetree.md

    -   `max_parts_in_total` — Maximum number of parts in all partitions.
	-   `max_compress_block_size` — Maximum size of blocks of uncompressed data before compressing for writing to a table. You can also specify this setting in the global settings (see [max_compress_block_size](../../../operations/settings/settings.md#max-compress-block-size) setting). The value specified when table is created overrides the global value for this setting.
	-   `min_compress_block_size` — Minimum size of blocks of uncompressed data required for compression when writing the next mark. You can also specify this setting in the global settings (see [min_compress_block_size](../../../operations/settings/settings.md#min-compress-block-size) setting). The value specified when table is created overrides the global value for this setting.

>     -   `max_partitions_to_read` — Limits the maximum number of partitions that can be accessed in one query. Can be overriden with query-level [max_partitions_to_read](../../../operations/settings/merge-tree-settings.md#max-partitions-to-read) setting.

Reviewer: ```suggestion
    -   `max_partitions_to_read` — Limits the maximum number of partitions that can be accessed in one query. You can also specify setting [max_partitions_to_read](../../../operations/settings/merge-tree-settings.md#max-partitions-to-read) in the global setting.
```
